compiler/modules/front.syntax/tests/data/testParser.json

```diff
@@ -0,0 +1,373 @@
+[
+    ["E", "expr", ""],
+    { "comment" : "not lexical errors" },
```

Comment:
This is a bit ugly: some of these are akin to section names; some apply to a particular test. But I guess it's not worth spending time on at this point.

Reply:
I thought about adding another level, and decided against it thinking everything might not neatly fall into categories.
I'll create an issue to consider in the future.